examples/rllib_traffic_corridor_2_teams.py

```diff
@@ -0,0 +1,130 @@
+
+import numpy as np
+
+from abmarl.managers import AllStepManager
+from abmarl.external import MultiAgentWrapper
+from abmarl.examples.sim.traffic_corridor import WallAgent, TargetAgent, TrafficAgent, \
+    TrafficCorridorSimulation
+
+grid_1_lane_2_teams = np.array([
+    ['G', 'W', 'W', 'W', 'R'],
+    ['r', '_', '_', '_', 'g'],
+    ['G', 'W', 'W', 'W', 'R'],
+])
+
+
+object_registry = {
+    'R': lambda n: TrafficAgent(
+        id=f'red{n}',
+        encoding=1,
+        render_color='red',
+    ),
+    'G': lambda n: TrafficAgent(
+        id=f'green{n}',
+        encoding=2,
+        render_color='green',
+    ),
+    'r': lambda n: TargetAgent(
+        id='red_target',
+        encoding=1,
+        render_color='red',
+        render_shape='s'
+    ),
+    'g': lambda n: TargetAgent(
+        id='green_target',
+        encoding=2,
+        render_color='green',
+        render_shape='s'
+    ),
+    'W': lambda n: WallAgent(
+        id=f'wall{n}',
+        encoding=3,
+        render_shape='s'
+    )
+}
+
+
+grid = grid_1_lane_2_teams
+sim = MultiAgentWrapper(
+    AllStepManager(
+        TrafficCorridorSimulation.build_sim_from_array(
+            grid,
+            object_registry,
+            overlapping={1: {1}, 2: {2}},
+            # TODO: Don't know the agent's id's beforehand, how to create better mapping?
```

Comment:
We talk about this in #394, but I think we need another ticket to handle the bigger picture, which is how to build simulations that require agent data when the agent data is not available until build time, such as when building from an array or a file.

Reply:
#476